packs/feats/animal-shape.json

```
@@ -24,7 +24,129 @@
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Dinosaur Form (Tyrannosaurus)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Dinosaur Form (Triceratops)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Dinosaur Form (Stegosaurus)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Dinosaur Form (Deinonychus)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Dinosaur Form (Brontosaurus)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Dinosaur Form (Ankylosaurus)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Insect Form (Spider)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Insect Form (Scorpion)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Insect Form (Mantis)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Insect Form (Centipede)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Insect Form (Beetle)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Insect Form (Ant)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Aerial Form (Wasp)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Aerial Form (Pterosaur)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Aerial Form (Bird)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Aerial Form (Bat)"
+                    }
+                ],
+                "flag": "beastkin-animal",
+                "key": "ChoiceSet"
+            },
+            {
+                "actorFlag": true,
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.UI.ChoiceSet.YesLabel",
+                        "value": 1
+                    },
+                    {
+                        "label": "PF2E.UI.ChoiceSet.NoLabel",
+                        "value": 0
+                    }
```

Missed `RuleElements` in between there
```suggestion
                    {
                        "label": "PF2E.UI.RuleElements.ChoiceSet.YesLabel",
                        "value": 1
                    },
                    {
                        "label": "PF2E.UI.RuleElements.ChoiceSet.NoLabel",
                        "value": 0
                    }
```